doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1478 @@
+
+* :ref:`ble_llpm` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+* :ref:`bluetooth_radio_coex_1wire_sample` sample:
+
+  * Added support for the ``nrf54h20dk/nrf54h20/cpurad`` and ``nrf54l15dk/nrf54l15/cpuapp`` build targets.
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* Updated:
+
+  * The values for the :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option in all configurations, and for the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_TX_POWER_CORRECTION_VAL` Kconfig option in configurations with the Find My Device Network (FMDN) extension support.
+    The values are now aligned with the Fast Pair requirements.
+  * The sample configurations to use a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`).
+    This is done to work around the timeout in MPSL flash synchronization (``NCSDK-29354`` known issue).
+    See :ref:`known_issues` for details.
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Added:
+
+    * LED indication on development kits for the Fast Pair advertising state.
+    * An application versioning using the :file:`VERSION` file.
+    * The DFU support which can be enabled using the ``SB_CONFIG_APP_DFU`` sysbuild Kconfig option.
+      DFU is available for all supported targets except the ``debug`` configurations of :ref:`zephyr:nrf52dk_nrf52832` and :ref:`zephyr:nrf52833dk_nrf52833` due to size constraints.
+
+  * Updated:
+
+    * The :ref:`ipc_radio` image configuration by splitting it into the debug and release configurations.
+    * The location of the sample configuration.
+      It has been moved from the root sample directory to the dedicated folder (:file:`locator_tag/configuration`).
+    * The ``fp_adv`` module to use the trigger requests for the Fast Pair advertising state instead of setting the Fast Pair advertising mode directly.
+
+Bluetooth Mesh samples
+----------------------
+
+* For all Bluetooth Mesh samples:
+
+  * Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board.
+  * Added support for Zephyr Memory Storage (ZMS) when compiling for the :ref:`zephyr:nrf54l15dk_nrf54l15` board.
+  * Removed support for the nRF54L15 PDK.
+
+* :ref:`bluetooth_ble_peripheral_lbs_coex` sample:
+
+  * Updated the usage of the :c:macro:`BT_LE_ADV_CONN` macro.
+    See the Bluetooth Host section in Zephyr's :ref:`zephyr:migration_3.7`.
+
+Cellular samples
+----------------
+
+* Added the :ref:`uicc_lwm2m_sample` sample.
+
+* :ref:`fmfu_smp_svr_sample` sample:
+
+  * Removed the unused :ref:`at_cmd_parser_readme` library.
+
+* :ref:`modem_shell_application` sample:
+
+  * Added ``link modem`` command for initializing and shutting down the modem.
+  * Updated to use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
+
+* :ref:`nrf_cloud_rest_fota` sample:
+
+  * Added support for setting the FOTA update check interval using the config section in the shadow.
```

Comment:
```suggestion
  * Added:

    * Support for setting the FOTA update check interval using the config section in the shadow.
    * Support for SMP FOTA on the nRF9160 DK.

```

from https://github.com/nrfconnect/sdk-nrf/pull/18145

Reply:
Added to https://github.com/nrfconnect/sdk-nrf/pull/18192/files#diff-edb5a676e90130a0ac595e21ff13ec34ea32ae0c53dad433bc0d0eab4d10030b file.